mealie/services/scheduler/tasks/create_timeline_events.py

```diff
@@ -19,7 +19,8 @@
 
 
 def create_mealplan_timeline_events(group_id: UUID4 | None = None):
-    event_time = datetime.now(timezone.utc)
+    # Move it back by 1 minute to ensure that midnight processing logs it to the prev day
```

Comment:
Is this following a precedent set somewhere else?
Personally I consider 00:00:00 on 27/10/2023 to _be on 27/10_. Sounds like you're wanting to say that's actually 26/10?

Ultimately consistency across the app is the most important thing, hence wondering if there's a precedent.

Reply:
The thought is that it's "end of day" processing--at midnight, it handles events for the previous day.
So since we're doing this at midnight, we want our event to happen at 23:59 to log it to the day that it was assigned to.